Makefile

```diff
@@ -193,8 +204,24 @@ test: env
 	@mkdir -p $(WORKING_DIRECTORY)/.tmp/home
 	@mkdir -p $(WORKING_DIRECTORY)/.tmp/home/certificate-authority
 	@mkdir -p $(WORKING_DIRECTORY)/.tmp/report
-	@$(call RUN-TESTS,hub,./...,"")
-	@$(call RUN-TESTS,grpc-gateway-dtls,./grpc-gateway/service,"TEST_COAP_GATEWAY_UDP_ENABLED=true")
+	@$(call RUN-TESTS,hub,./...,-timeout=45m -race -p 1 -v -tags=test,"")
```

Comment:
This will only invoke files with the `test` tag, and not any files with the `test_mem` or any files without any tags?

Reply:
From the logs it seems that files without tags get always invoked

Reply:
yes - it invokes only mentioned tags + without tags